anago

```diff
@@ -954,7 +954,7 @@ announce () {
   local arg="$1"
   local announcement_file="$WORKDIR/announcement.html"
   local subject_file="$WORKDIR/announcement-subject.txt"
-  local nomock_flag
+  local nomock_flag=""
```

Comment:
This is not necessary, but has the same net effect.

Reply:
yep, it was just to improve consistency with other parts of the program